docs/project/commit_access.md

```diff
@@ -0,0 +1,64 @@
+# Commit access
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+<!-- toc -->
+
+## Table of contents
+
+-   [Overview](#overview)
+-   [Getting access](#getting-access)
+-   [Removing access](#removing-access)
+
+<!-- tocstop -->
+
+## Overview
+
+Commit access is what allows developers to merge PRs into the main
+[carbon-lang repository](https://github.com/carbon-language/carbon-lang/). It
+also gives the ability to [review, approve, and merge changes](code_review.md).
```

Comment:
One larger and one smaller thing here...

First, I'd like to emphasize that commit access on Carbon is *not* required for contribution, and really encourage folks to contribute even without it. I've tried to add a paragraph to this effect, but happy for you to rewrite to suit.

A second small thing is that I think reviewing works even w/o commit access, so it's really just approving and merging?

```suggestion
First and foremost, commit access is **not required** for contributing to Carbon! Anyone can send a pull request with a change to Carbon. Very few parts of our process require commit access, and most development activity is exactly the same for folks with or without. This is an important feature of our GitHub workflow for us, and one we plan to keep.

The main thing commit access allows is for developers to merge PRs into the main
[carbon-lang repository](https://github.com/carbon-language/carbon-lang/). They still need review, and often a reviewer with commit access handles the merge. It
also gives the ability to [approve and merge changes](code_review.md).
```